src/components/map/StopMarker.js

```diff
@@ -19,68 +22,87 @@ export default ({
   onChangeShowTime,
   onTimeClick,
 }) => {
+  const isTerminal = firstTerminal || lastTerminal;
+
   const timingStopIcon = icon({
     iconUrl: TimingStopIcon,
-    iconSize: [20, 20],
-    iconAnchor: [15, 20 / 2],
+    iconSize: [30, 30],
+    iconAnchor: [23, 25 / 2],
     popupAnchor: [3, -15],
+    className: "stop-marker timing-stop",
   });
 
+  let journeyStartedOnTime;
+
+  if (firstTerminal && hfp.length === 1) {
+    const date = time.format("YYYY-MM-DD");
+    const journeyStartHfp = get(reverse(hfp), `[0].journeys[0].depart`, "");
+
+    const startedDate = new Date(journeyStartHfp.receivedAt);
+    const journeyStartDate = new Date(`${date}T${journeyStartHfp.journeyStartTime}`);
+
+    journeyStartedOnTime = Math.abs(diffDates(startedDate, journeyStartDate)) <= 60;
+  }
+
```

Comment:
This has to do with coloring the origin terminal if the journey was started over 60 seconds late or early. It will only work if a journey is selected, otherwise the stop is dealing with times from multiple journeys and showing a color becomes illogical.